src/hu_stuff.c

```diff
@@ -991,6 +991,37 @@ static void HU_widget_build_keys (void)
     hud_keysstr[i++] = ' ';   // spacing
     hud_keysstr[i++] = ' ';
   }
+
+  // [Alaux] Blink missing keys *after* possessed keys
```

Comment:
One last thing. In the Boom HUD the card keys are shown first and the skull keys second. Now, if you already have a skull key, but need a card key then the blinking card key will be shown after the skull key and not where it belongs.

Reply:
That's intentional. With the previous approach, the possessed keys would get mixed up with the missing ones, and it looked confusing to me. Tweaking the current approach so missing cards and skulls are drawn with possessed cards and skulls respectively would partially bring back that quirk.

Reply:
Alright, understood.